skimage/registration/__init__.py

```diff
@@ -1,7 +1,15 @@
 from ._optical_flow import optical_flow_tvl1
 from ._phase_cross_correlation import phase_cross_correlation
 
+from ._lddmm import diffeomorphic_metric_mapping
+from ._lddmm_utilities import resample, sinc_resample, generate_position_field
+
+
 __all__ = [
     'optical_flow_tvl1',
-    'phase_cross_correlation'
+    'phase_cross_correlation',
+    'lddmm_register',
+    'resample',
```

Comment:
That sort of depends on whether they are desirable. They all provide functionality that, as far as I can tell, don't exist elsewhere. However, none of them are separately necessary to use diffeomorphic_metric_mapping.
- resample is basically the same functionality as resize/rescale but with slightly different behavior, i.e. resize takes an image and resamples it to a given shape, rescale takes an image and a scale, rounds the scale, and calls resize. resample takes an image, its resolution, and a desired resolution, and resamples it to exactly that desired resolution. You can get the same behavior from resize/rescale only with particular resolutions and image shapes.
- sinc_resample is smooth fourier-domain resampling function appropriate for arrays with certain properties.
- generate_position_field is an lddmm-relevant utility function for advanced users, it accepts a velocity-fields array and returns a position-field array. velocity_fields is an internal attribute containing the deformative information of the transformation in diffeomorphic_metric_mapping and an initial value may be provided for it. The ultimate transformations that are returned and used to deform images are position-fields. This function exists so that users can manipulate a velocity-fields object, possibly the one returned by diffeomorphic_metric_mapping in the internals section, or perhaps their own.